ci-operator/jobs/openshift/cloud-network-config-controller/openshift-cloud-network-config-controller-master-presubmits.yaml

```diff
@@ -146,6 +146,80 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-azure,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
I would suggest `always_run: false` in this job and all others.

Azure stack tests have not reached stability and capacity is extremely low

Reply:
if this is the case, we may want `optional: true` as well. that goes in the config file and will be replicated to this jobs file when you do `make jobs`

Reply:
Yes this PR is already correct to set it to `optional: true` in the config. It must be optional, because azurestack-e2e has not yet passed e2e-tests

Reply:
I tried to add `always_run: false` here but it was removed in a `make update`. What am I missing here?